autotest-external.sh

```diff
@@ -31,8 +31,13 @@ function print_syntax {
 }
 
 if ! [ -x "$PHPUNIT" ]; then
-	echo "phpunit executable not found, please install phpunit version >= 4.8" >&2
-	exit 3
+	echo "phpunit executable not found, trying local one from build/integration" >&2
+	if [ -x "$PWD/build/integration/vendor/phpunit/phpunit/phpunit" ]; then
+		PHPUNIT="$PWD/build/integration/vendor/phpunit/phpunit/phpunit"
+	else
+		echo "phpunit executable not found, please install phpunit version >= 4.8" >&2
```

Comment:
Bump this
```suggestion
		echo "phpunit executable not found, please install phpunit version >= 9.6" >&2
```

4.8 was for PHP 5.x support...

Reply:
Not changed but yes can do so

Reply:
@nickvergessen this breaks Drone as drone uses PHPUnit 9.4.4 not 9.6.0